doc/source/user/absolute-beginners-outline.rst

```diff
@@ -0,0 +1,137 @@
+**************
+NumPy: The Absolute Basics for Beginners
+**************
+
+This is a working outline for a future section introducing NumPy to absolute beginners. If you have comments or suggestions, please don’t hesitate to reach out!
+
+
+
+- How to install NumPy
+  
+  - various operating systems 
+
+- How to import NumPy
+
+- What is an array
+  
+  - What else might it be called
+  
+  - What is its shape
+
+  - Can you reshape it?
+
+  - What’s the difference between a Python List and a NumPy array? Why use NumPy?
+
+  - What are the attributes of an array?
+
+  - Broadcasting example
+
+- How to create an array (ndarray object)
+- How to create a basic array
+- How to create an array from existing data
+- How to create a new array from an existing array
+- How to specify the datatype
+  
+  - Examples of commonly used NumPy dtypes
+
+- How to inspect the size and shape of a NumPy array
+- How to check whether a list is empty or not
+- How to represent missing values and infinite values
+- Indexing and Slicing
+- Basic array operations (np.sum, np.dot, np.append, np.diff
+
+  - Operations on a single array
+
+  - Unary operators
+
+  - Binary operators
+
+  - Universal functions
+
+- How to compute mean, minimum, maximum, cumulative sum
+  
+  - (include row-wise and column-wise compute)
+
+- How to concatenate two arrays
+  
+  - column-wise
+
+  - row-wise
+
+    - np.concatenate, np.vstack, np.hstack, np.r_, np.c_
```

Comment:
there's also `np.stack`. please leave out `r_` and `c_`, they shouldn't really be used/preferred

Reply:
will do!